docs/source/conf.py

```diff
@@ -44,6 +44,10 @@
     #'sphinx.ext.graphviz',
 ]
 
+# Adding the github files extension
+sys.path.append(os.path.abspath("./_ext"))
```

Comment:
perhaps this should be `os.path.join(".", "_ext")` ?

Reply:
Yes, that works.